docs/UserGuide.md

```diff
@@ -62,16 +67,64 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+* There are fixed multiple valid date and time formats (GMT+8 24-Hour Time format):
+  1. dd/MM/yyyy HH:mm (e.g. 15/09/2022 13:00)
+  2. dd-MM-yyyy HH:mm (e.g. 15-09-2022 13:00)
+  3. yyyy-MM-dd-HH-mm (e.g. 2022-09-15-13-00)
 </div>
 
 ### Viewing help : `help`
 
-Shows a message explaning how to access the help page.
+Shows a message explaining how to access the help page.
 
 ![help message](images/helpMessage.png)
 
 Format: `help`
 
+### Adding a new activity: `add t/activity`
+
+Adds a new activity into your GoMedic scheduler. 
+
+Format: `add t/activity s/START_TIME e/END_TIME ti/TITLE [d/DESCRIPTION]`
+* `START_TIME` and `END_TIME` must follow one of the formats specified. 
+* `START_TIME` is strictly less than `END_TIME`.
+* Clashing activity (including partial overlap with other activities or appointments) would be considered as invalid activity (i.e. not to be added).
+* `TITLE` ideally should be very short so that it can be displayed in the list without being truncated.
+
+**Tip:** Try to pack as many keywords as possible inside the description while keep it concise so that you can search it later. 
+
+Examples:
+* `add t/activity s/2022-09-15-14-00 e/15/09/2022 15:00 ti/Meeting with Mr. X d/about a certain paper`
+* `add t/activity s/15/09/2022 14:00 e/15/09/2022 15:00 ti/Meeting with Mr. Y`
+
+### Deleting an existing activity: `delete t/activity`
+
+Delete a certain existing activity 
+
+Format: `delete t/activity i/ACTIVITY_ID`
+* Activity ID can be obtained by listing all the activities or search for a certain activities within a certain time frame. 
+* Activity ID is **unique** (i.e. every activity will be assigned to a unique ID, hence this guarantees 1 `delete t/activity` command will not delete 2 activities at once).
+* Invalid Activity ID being supplied would be flagged by GoMedic, and do not cause changes to any existing activities.
+
+Examples: 
+* `delete t/activity A123`
+
+### List all activities: `list t/activity`
+
+List all existing (past, present and future) activities that exist in GoMedic. 
+
+Format: `list t/activity`
+* Activities would be displayed in ascending `START_TIME` (The past activities would be at the top if any and Future activities at the bottom).
+* The summary or description that are too long would be truncated. 
+* The `START_TIME` and `END_TIME` would be displayed in `dd-MM-yyyy HH:mm` GMT+8 24-Hour format.
+
+<div markdown="span" class="alert alert-primary">
+:bulb: **Tip:** There are other upcoming `list` commands that can list future activities only and past activities only.
+</div>
```

Comment:
Seems like the `<div markdown="span" ...` format only works for certain sites that support this syntax, e.g. GitHub Pages. Should we change it to a more general HTML syntax like
```
<div class="alert alert-primary">
:bulb: <strong>Tip:</strong> There are other upcoming <code>list</code> commands that can list future activities only and past activities only.
</div>
``` 
or even remove the HTML block, so that the correct rendering is not only limited to sites which support the flavour of syntax in line 121?

See e.g. of how it is rendered in GitHub (not GitHub Pages):
![image](https://user-images.githubusercontent.com/48285332/134623572-e9c42dd2-0a7a-42f4-9a90-a6f17c4fe8cb.png)

Reply:
Yeah sure I think we can just remove it